R/add_family_name_column.R

```diff
@@ -0,0 +1,26 @@
+#' Adds a column with the name of the language family.
+#'
+#' @param Table data-frame of CLDF table with the columns  "Family_ID", "Name" and "Glottocode".
+#' @return data-frame with Family_name column.
+#' @note It is necessary that for every unique glottocode in Family_ID there is a row with a Glottocode and Name to match that. If there isn't, languages will have missing values for their Family_name even though they are not isolates.
```

Comment:
It should be mentioned that this assumption is met by Glottolog CLDF's LanguageTable - and probably by not too many other datasets.

Reply:
Stimm. I added this to the Note and one of the error messages already.

Reply:
I'll add it to the error message in this one too

Reply:
I think I've added this all place necessary now. Thanks for noting that @xrotwang !